.github/workflows/pr-realm-js.yml

```diff
@@ -713,6 +734,17 @@ jobs:
           arch: x86
           ndk: ${{ env.NDK_VERSION }}
           cmake: 3.22.1
-          working-directory: integration-tests/environments/react-native-test-app
-          script: npx mocha-remote -- concurrently --kill-others-on-fail npm:metro npm:runner
-          
+          working-directory: integration-tests/environments/react-native-test-app/android
+          script: |
+            # Setup port forwarding to Mocha Remote
+            adb reverse tcp:8090 tcp:8090
+            # Uninstall the app if already in the snapshot (unlikely but could result in a signature mismatch failure)
+            adb uninstall com.microsoft.reacttestapp || true
```

Comment:
Uninstalling here should fix the issues we're having with the Android test app being unable to install on other branches: https://github.com/realm/realm-js/actions/runs/9680928508/job/26757061658?pr=6743#step:16:526
> Unable to install /Users/runner/work/realm-js/realm-js/integration-tests/environments/react-native-test-app/android/app/build/outputs/apk/debug/app-debug.apk
[runner] com.android.ddmlib.InstallException: INSTALL_FAILED_UPDATE_INCOMPATIBLE: Package com.microsoft.reacttestapp signatures do not match previously installed version; ignoring!